test/hotspot/jtreg/runtime/stringtable/StringTableCleaningTest.java

```diff
@@ -59,6 +59,8 @@ public static void main(String[] args) throws Exception {
         subargs.add(Tester.class.getName());
         subargs.addAll(Arrays.asList(args));
         OutputAnalyzer output = ProcessTools.executeTestJvm(subargs);
+        output.outputTo(System.out);
+        output.errorTo(System.out);
```

Comment:
The normal way to do this is to call output.reportDiagnosticSummary(). But you only do that after the actual checks, when you know you have success, as in the error cases it will be called anyway and you don't want the output twice.

Reply:
Hmm, I see now that adding lines those is unnecessary, so I'll just remove them. Both shouldHaveExitValue() and checkOutput() seem to do the right thing. Not sure why I thought they didn't.